GliaWidgets/ViewModel/EngagementViewModel.swift

```diff
@@ -50,6 +50,7 @@ class EngagementViewModel {
     private let screenShareHandler: ScreenShareHandler
     private(set) var isViewActive = ObservableValue<Bool>(with: false)
     private static var alertPresenters = Set<EngagementViewModel>()
+    private var isEngagementEnded = false
```

Comment:
Maybe just to clarify that engagement ended on operator side?
isEngagementEndedByOperator ?

Reply:
this var for the case when Visitor ends Engagement. so that we don't need to notify that operator ended engagement